nestkernel/conn_parameter.cpp

```diff
@@ -39,7 +39,7 @@
 nest::ConnParameter*
 nest::ConnParameter::create( const Token& t, const size_t nthreads )
 {
-  // Code grabbed from TopologyModule::create_parameter()
+  // Code grabbed from NestModule::create_parameter()
   // See there for a more general solution
```

Comment:
Are these two lines needed anymore?

Reply:
@stinebuu: You're right. Removed in 409b2803e.